node_modules/oae-core/groupprofile/bundles/default.properties

```diff
@@ -0,0 +1,3 @@
+ADD_A_DESCRIPTION_TO_THIS_GROUP = Add a description to this group
+NO_DESCRIPTION_PROVIDED_YET = No description provided yet
```

Comment:
I would recommend not to keep that header, only because "About" doesn't make sense if only member thumbnails display.